### PR TITLE
Add test: Milestone 6

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this Milestone the following was done:

- Set up React Testing Library.
- Followed the instructions in the official React [documentation](https://testing-library.com/docs/react-testing-library/intro/).
- Wrote unit tests for the files operate.js and calculate.js using Jest.
- Created unit tests for all React components:
- Used Jest snapshots to test the components.
- Used React Testing Library to simulate user interaction.
- Ran full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,4 +80,3 @@ Give a :star: if you like this project!
 ## 📝 License
```
Comment (Dipeshtwis) [inline README.md:147]: - [x] README file should have steps about how to test the project. (Hint: `npm run test`)

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pasytchangwa and @sheylaPozo, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: All the functionalities & scenarios of the operate.js modules are covered using Jest.
:+1: There are Jest snapshots for testing all the components.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] I would recommend adding some React Testing Library to simulate user interaction.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (pasytchangwa) [reply README.md:147]: Done

**Review by hemantso — APPROVED** [review]
Hi @pasytchangwa and @sheylaPozo , 

## Your project is complete!
 There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**